src/Models/BaseElement.php

```diff
@@ -443,15 +442,29 @@ public function getRenderTemplates($suffix = '')
                 break;
             }
 
+            $namespacelessClassname = $this->stripNamespacing($value);
+
             $templates[] = $value . $suffix;
-            $templates[] = 'elements/' . DataObjectPreviewController::stripNamespacing($value) . $suffix;
-            $templates[] = DataObjectPreviewController::stripNamespacing($value) . $suffix;
+            $templates[] = 'elements/' . $namespacelessClassname . $suffix;
```

Comment:
@wilr has removed this line in #113, whichever doesn't get merged first will need to rebase anyway, but we can remove this line because templates are namespaced now anyway. In fact - this logic isn't really applicable any more, since we _do_ want namespaced classes when looking up template names